nf_core/utils.py

```diff
@@ -965,13 +965,13 @@ def load_tools_config(directory="."):
 
 
 def determine_base_dir(directory="."):
-    base_dir = Path(directory).absolute()
+    base_dir = start_dir = Path(directory).absolute()
```

Comment:
```suggestion
    base_dir = Path(directory).absolute()
```
`base_dir` and `start_dir` are the same

Reply:
`base_dir` can get modified in the loop. The idea was to be able to get the relative path at the end, but it will always be `.` if `start_dir` contains the `.nf-core.yml`. To be honest I think logging was always broken if the while loop got entered but it just rarely did.

Reply:
ah ok I get it now!